h/models/job.py

```diff
@@ -0,0 +1,32 @@
+from sqlalchemy import Column, DateTime, UnicodeText, func, text
+from sqlalchemy.dialects.postgresql import JSONB, UUID
+
+from h.db import Base
+
+
+class Job(Base):
+    """A job in the job queue. See services/job_queue.py for more details."""
+
+    __tablename__ = "job"
+
+    id = Column(UUID, server_default=func.uuid_generate_v1mc(), primary_key=True)
+    enqueued_at = Column(DateTime, nullable=False, server_default=func.now())
+    scheduled_at = Column(DateTime, nullable=False, server_default=func.now())
+    tag = Column(UnicodeText, nullable=False)
+    kwargs = Column(
+        JSONB,
+        server_default=text("'{}'::jsonb"),
+        nullable=False,
+    )
```

Comment:
`tag` is gonna be used to distinguish jobs added by admin pages from those added by the API. For example if someone uses an admin page to add a lot of jobs to the queue and we want to remove them, this will make it easy to delete only the jobs added by the admin page.

`scheduled_at` is gonna be used by the API to schedule jobs not to be processed until a minute after the API call. (Whereas jobs scheduled by admin pages can be processed immediately.)

`enqueued_at` is for sorting.

Reply:
I think we might want to add a `type` column (string) even though, currently, all rows would have the value `"sync_annotation"`. I think a `type` column is the one thing that's missing that would be needed to potentially reuse the job queue for a second type of job in the future, without having to do a data migration to add a `type` to existing rows.